collect_app/src/main/java/org/odk/collect/android/utilities/ImageConverter.java

```diff
@@ -76,7 +76,7 @@ private static Integer getMaxPixelsFromFormIfDefined(QuestionWidget questionWidg
 
     private static Integer getMaxPixelsFromSettings(Context context) {
         Integer maxPixels = null;
-        String imageSizeMode = (String) GeneralSharedPreferences.getInstance().get(KEY_IMAGE_SIZE);
+        String imageSizeMode = (String) new GeneralSharedPreferences(context).get(KEY_IMAGE_SIZE);
```

Comment:
Should this not be Injectable?

Reply:
This is a static method, so the code needs to be refactored first.

Reply:
Of course yes 👍

Reply:
Done